3.2/Dockerfile

```diff
@@ -63,13 +64,16 @@ ENV BASH_ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     ENV=${CONTAINER_SCRIPTS_PATH}/scl_enable \
     PROMPT_COMMAND=". ${CONTAINER_SCRIPTS_PATH}/scl_enable"
 
+COPY s2i/bin $STI_SCRIPTS_PATH
 COPY root /
 
 # this is needed due to issues with squash
 # when this directory gets rm'd by the container-setup
 # script.
 RUN /usr/libexec/container-setup
 
+RUN usermod -a -G root redis
```

Comment:
Is there any `redis` user?

Reply:
yes there is, we renamed [here](https://github.com/sclorg/redis-container/pull/34/files#diff-c1f8cc5d9377fe498f7178c8b7a322beR40) `default` user who is beeing created in s2i-base
we also need `root` group because of fix-permission script
but as I see now, in s2i-base there is `default` user created with group 0 then its default group gets changed to redis and then added to root group. Yes a bit confusing.

Reply:
Thanks. I've missed it.